### PR TITLE
Finalize Deployment script and ARM templates, and add README 

### PR DESCRIPTION
**Pull request by rabee333**
@amiturgman 
I merged the 2 types of deployments (WebApp and ASE) into one script, but still with separated ARMs.
Added a README file how to deploy in Azure.

### CODE REVIEW

File: azure-deployment/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Azure Deployment script
+PowerShell script for deploying ARM templates to create the Supply Cahin environment in Azure.
```
Comment (amiturgman) [inline azure-deployment/README.md]: typo in 'Chain'
environment ==> solution

File: azure-deployment/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Pre Requisites
+To run the deployment script, please make sure you have the following pre-requisites:
+1.	PowerShell to run the script.
```
Comment (amiturgman) [inline azure-deployment/README.md]: no need to end the bullets with a dot '.'

File: azure-deployment/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+2.	Node v8.4 or later.
+3.	Azure subscription.
+
+## Getting Started
+Before running the script open the `deploy.conf` file and set the script configuration values.
+Here is a brief description of the configuration values:
+
+### [General]
+* `deploymentType`: This is the deployment type. I could be one of two types:
```
Comment (amiturgman) [inline azure-deployment/README.md]: **It** can be...

File: azure-deployment/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    1.	WebApp: Use this option to deploy less secured environment. This option is used generally for development purposes.
```
Comment (amiturgman) [inline azure-deployment/README.md]: WebApp: This is an environment for development and preview purposes. It is less secured and should not be used for production purposes.

File: azure-deployment/README.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    2.	ASE: Use this option to deploy a secured environment with App Service Environment. Note: this option costs more that the WebApp option, but it’s strongly recommended for production environments.
```
Comment (amiturgman) [inline azure-deployment/README.md]: more **than**

Comment (amiturgman) [reply azure-deployment/README.md]: .. but is recommended for production purposes

File: azure-deployment/README.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+* `namePrefix`: The name prefix of all the resources created by this script, except the Network Interfaces and the Storage Accounts of the Ethereum Consortium resources.
+* `resourceGroupName`: The Resource Group name that will include all the created resources by this script.
+* `resourceGroupLocation`: The geographic location of the resource group. Please choose one of the locations listed [here](https://azure.microsoft.com/en-us/regions/)
+* `deploymentName`: The name of the deployment.
+* `gitSmartContractsRepoURL`: The Smart Contracts repository git URL. Default value: `https://raw.githubusercontent.com/CatalystCode/ibera-supply-chain/`
```
Comment (amiturgman) [inline azure-deployment/README.md:22]: keep in mind that we have a task to remove all **iBera** occurrences from all the repo names...
This should be changed here and everywhere once we do.

File: azure-deployment/README.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+* `gitSmartContractsFolder`: The folder name where the smart contracts repository will be cloned. Default value: `ibera-smart-contracts`
+* `mainRepositoryLocation`: The location URL where is the resources of the main repository (the Supply Chain umbrella) are available for download. Default value: `https://raw.githubusercontent.com/CatalystCode/ibera-supply-chain/`
+* `mainRepositoryBranch`: The used branch of the main repository. Default value: `master`
+
+### [Etherium Consertium]
```
Comment (amiturgman) [inline azure-deployment/README.md]: Consortium

File: azure-deployment/README.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+* `oiName`: The name of the Office Integration web app. Default value: `oiapi`
+* `servicesStorageAccountName`: The name of the storage account used by the Services web app. Default value: `svcstorage`
+* `oiStorageAccountName`: The name of the storage account used by the Office Integration web app. Default value: `oistorage`
+* `gitServicesRepoURL`: The Services repository git URL. Default value: `https://github.com/CatalystCode/ibera-services.git`
+* `gitServicesBranch`: The used branch of the Services repository. Default value: `master`
+* `gitOiRepoURL`: The Office Integration repository git URL. Default value: `https://github.com/CatalystCode/ibera-office-integration.git`
+* `gitOiBranch`: The used branch of the Office Integration repository. Default value: `master`
+
+## Running the script
+To deploy the environment in Azure, make sure you have the pre-requisites listed in the beginning of this document, and make sure you set the needed deployment parameters in the `deploy.conf` file, and then follow these steps:
```
Comment (amiturgman) [inline azure-deployment/README.md]: To deploy the **solution**

Comment (amiturgman) [reply azure-deployment/README.md]: needed ==> required

File: azure-deployment/README.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
+# Azure Deployment script
```
Comment (amiturgman) [inline azure-deployment/README.md:1]: Add a reference to this README file in the main README file (the one that's on the root). Under deployment section there should be a link to this folder.

In the end of this file add a comment to go back to the main README file (the on in the root).

Comment (rabee333) [reply azure-deployment/README.md:1]: done

File: azure-deployment/README.md (hunk 1 above)
Comment (rabee333) [reply azure-deployment/README.md]: fixed

File: azure-deployment/README.md (hunk 2 above)
Comment (rabee333) [reply azure-deployment/README.md]: removed

File: azure-deployment/README.md (hunk 3 above)
Comment (rabee333) [reply azure-deployment/README.md]: fixed

File: azure-deployment/README.md (hunk 4 above)
Comment (rabee333) [reply azure-deployment/README.md]: changed

File: azure-deployment/README.md (hunk 5 above)
Comment (rabee333) [reply azure-deployment/README.md]: changed

File: azure-deployment/README.md (hunk 6 above)
Comment (rabee333) [reply azure-deployment/README.md:22]: yes sure

File: azure-deployment/README.md (hunk 7 above)
Comment (rabee333) [reply azure-deployment/README.md]: fixed

File: azure-deployment/README.md (hunk 8 above)
Comment (rabee333) [reply azure-deployment/README.md]: changed

**Review by amiturgman — APPROVED** [review]

**Comment by amiturgman** [comment]
@rabee333 looks like I don't have admin permissions so I can't merge.
I asked @cloudbeatsch  to add me to give me permissions so I hope I will be able to merge soon.

**Pull request merged**